client/shipping-label/views/steps/packages/list.js

```diff
@@ -1,44 +1,65 @@
 import React, { PropTypes } from 'react';
-import NumberInput from 'components/number-field/number-input';
-import Gridicon from 'components/gridicon';
-import { sanitize } from 'dompurify';
-
-const renderItemInfo = ( item, itemIndex ) => (
-	<li key={ itemIndex }>
-		<span dangerouslySetInnerHTML={ { __html: sanitize( item.name ) } } />
-		{ 1 < item.quantity ? ' (x' + item.quantity + ')' : null }
-	</li>
-);
+import { translate as __ } from 'lib/mixins/i18n';
+import NumberField from 'components/number-field';
+import FormLegend from 'components/forms/form-legend';
+import { sprintf } from 'sprintf-js';
 
 const renderPackageDimensions = ( pckg, dimensionUnit ) => {
 	return `${pckg.length} ${dimensionUnit} x ${pckg.width} ${dimensionUnit} x ${pckg.height} ${dimensionUnit}`;
 };
 
 const OrderPackages = ( { packages, updateWeight, dimensionUnit, weightUnit, errors } ) => {
+	const renderItemInfo = ( item, itemIndex ) => {
+		return (
+			<div key={ itemIndex } className="wcc-package-item">
+				<div className="item-name">
```

Comment:
I think this className is dangerously generic. Let's do something like wcc-package-item__name or something